cmd/argo/commands/list.go

```diff
@@ -112,6 +126,7 @@ func NewListCommand() *cobra.Command {
 		},
 	}
 	command.Flags().BoolVar(&listArgs.allNamespaces, "all-namespaces", false, "Show workflows from all namespaces")
+	command.Flags().StringVar(&listArgs.prefix, "prefix", "", "Show only workflows with names has prefix")
```

Comment:
"Filter workflows by prefix" should be better here

Reply:
modified.